### ISSUE TITLE
[PD] Regular Study Sessions

### ISSUE THREAD

**Issue by Priscilla-MacGatus** [Priscilla-MacGatus/Coursework-Planner#85]
**[<img src="https://avatars.githubusercontent.com/u/42203406?s=17&v=4" width="17" height="17"> @SallyMcGrath](https://github.com/SallyMcGrath)** cloned issue [Migracode-Barcelona/Module-JS1#16](https://github.com/Migracode-Barcelona/Module-JS1/issues/16) on 2024-07-04: 

> ### Coursework content
> 
> Organise a study session with your group. Use this session to work through your Personal Development coursework as a team. 
> 
> 1. Set up the invite on Google Calendar
> 
> 2. Define the agenda (what aspects of the PD lessons or coursework you will be covering) 
> 
> 3. Have the session using either Google Meet or Slack.
> 
> This must be done before the next class. 
> 
> 
> 
> ### Estimated time in hours
> 
> 0.5
> 
> 
> 
> ### What is the purpose of this assignment?
> 
> The aim is to have one study session with a group at least once every module, but you can decide as a group how often you want to have it recur. 
> 
> Once you've decided on the periodicity after your first session, edit your calendar invite to a recurring meeting. 
> 
> 
> 
> ### How to submit
> 
> - Add a screenshot of the calendar invite as a comment to this ticket
> 
> - Add a comment summarising how your first session went and what you discussed
> 
> 
> 
> ### Anything else?
> 
> Use this [video](https://www.youtube.com/watch?v=ZEXO5DHigi8) to learn how to create a calendar invite.

**Comment by Priscilla-MacGatus** [comment]
![Image](https://github.com/user-attachments/assets/e95bb97d-15ad-407d-ae02-42f12d26c7f2)

**Comment by Priscilla-MacGatus** [comment]
In our first session, we covered the fundamentals of JavaScript, including variable declaration with let, const, and var, as well as data types like strings, numbers, arrays, and objects. We discussed control structures such as conditionals and loops, and introduced functions, emphasizing their definition, invocation, and the concept of scope. Overall, the session provided a solid foundation for understanding JavaScript.
